projects/1.20/assets/goety-gotta-summon-em-all/vividerusmoregoetysummons/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "item.vividerusmoregoetysummons.necromancer_focus.info": "召唤死灵法师的仆从。潜行施法时，将使施法者的仆从向其靠拢",
+  "config.jade.plugin_vividerusmoregoetysummons.ally_servant": "",
+  "item.vividerusmoregoetysummons.thrashing_focus.info": "召唤重击者仆从。潜行施法时，将施法者的仆从向其靠拢",
+  "item.vividerusmoregoetysummons.lords_felsteed_focus.info": "召唤一位伯爵的恶魔马侍从。潜行施法时，将施法者的仆从向其靠拢",
+  "entity.vividerusmoregoetysummons.servant_zombie_fisherman": "僵尸渔夫仆从",
+  "item.vividerusmoregoetysummons.fallen_knight_focus": "混沌堕落骑士聚晶",
```

原文为Fallen Knight，并未出现Chaos，第9、31行同理
```suggestion
  "item.vividerusmoregoetysummons.fallen_knight_focus": "堕落骑士聚晶",
```